doc/reference/devices_gpu.md

```diff
@@ -53,6 +53,32 @@ Add a specific GPU from the host system as a `physical` GPU device to an instanc
 
 See {ref}`instances-configure-devices` for more information.
 
+### Configuration examples with the Container Device Interface (CDI) mode
```

Comment:
```suggestion
#### CDI mode
```

Reply:
We're already in the config example section, so let's just add a subsection instead of a second config examples section.

Reply:
fixed